### DISCUSSION TITLE
Error received while trying to chip and train

### DISCUSSION THREAD

**Discussion by georgeper2001** [azavea/raster-vision#2121] [General]
I ran the following commands which is using Object detection: first: "docker run -it --rm -v $HOME/.aws:/root/.aws:ro -v $(pwd):/workdir -w /workdir quay.io/azavea/raster-vision:pytorch-0.20 /bin/bash"
then: "rastervision run local chev_object_tst.py -a root_uri /tmp/hello -a cowc cowc.csv -a batch_size 16 -a epochs 32 -a arch FasterRCNN analyze chip train"
I got the following error: "RuntimeError: DataLoader worker (pid 611) is killed by signal: Bus error. It is possible that dataloader's workers are out of shared memory. Please try to raise your shared memory limit.
make: *** [/tmp/hello/Makefile:12: 2] Error 1"
Any idea why?

**Comment by AdeelH** [comment]
You will need to pass the `--ipc=host` option to `docker run`. Example:

https://github.com/azavea/raster-vision/blob/db46a7eaad15caea7d5e2002d9348bbb6a504351/docker/run#L141-L142
